models/engine/file_storage.py

```diff
@@ -68,3 +68,12 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
```

Comment:
Per our discussion on Slack, all of this file's changes should not be included in the PR, since it reverts approved changes to master.